### DISCUSSION TITLE
Why don't I see contributions on my GitHub page?

### DISCUSSION THREAD

**Discussion by rerarname** [community/community#18588] [Other Feature Feedback, Questions, & Ideas] [answered]
Why don't I see contributions on my GitHub page?

**Comment by epistax1s** [comment, marked as answer]
You can read about how contributions are calculated [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile). Please note that in some cases it may take up to 24 hours for the contributions to be displayed.
